### DISCUSSION TITLE
Einspeisebegrenzung mit 2 Hoymiles HMT 2250

### DISCUSSION THREAD

**Discussion by Mannipfan** [hoylabs/OpenDTU-OnBattery#1042] [Q&A]
Liebe Community, ich bin erst seit kurzem in diese Materie eingetaucht und bitte um Nachsicht etwaiger naiver Fragen.
Anlage in Betrieb. Hoymiles HMT 2250 MIT 6 SUNENERGY Modulen, OpenDTU on Battery, Shelly 3em, KEIN Baterriesystem! Einspeisevertrag bis 2.250 w.
Nun möchte ich das System mit dem gleichen Set ausbauen (HMT 2250 + 4 Module zu der südlich ausgerichteten Anlage, 2 Module Westseite) um an nicht optimalen Tagen bzw. am Abend mehr Leistung abrufen zu können. Nun zu meiner Frage:
Ist es möglich die Gesamtleistung beider kombinierten Systeme auf 2.250 w zu begrenzen? Ich habe schon heraus gelesen das die OpenDTU ON battery nur einen Wechselrichter steuern kann. Meine Überlegung ist daher einen WR unlimitiert (also bis max. 2.250 w) zu lassen und den zweiten WR über die DPL dynamisch zu regeln um bei schlechteren Bedingungen die restliche Leistung beizusteuern. Die Eingabemaske der DPL überfordert mich momentan noch ein wenig ;)
Schöne Grüße aus Österreich
Manfred

**Comment by genmad** [comment]
Mit der rechtlichen Komponente deiner Idee möchte ich mich nicht beschäftigen. Technisch könnte man es so probieren: 

Lass den ersten Wechselrichter frei laufen. Stell beim 2. Wr deine gewollte Abgabe ins Netz ein ( -2250W)  und schon sollte es funktionieren, dass du nicht mehr als 2250W einspeist.

Vorraussetzung: beide Wr‘s sind hinter dem gleichen Leistungsmesser und dieser ist in odob für Wr 2 konfiguriert. Ausserdem darfst du kein Solar Pass Through anschalten.

**Reply by Mannipfan** [reply]
Ich danke dir!
